packages/protocol/schemaCodegen.cjs

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
+
+const { exit } = require('process');
+const fs = require('fs');
+const tsGen = require('json-schema-to-typescript');
+const http = require('https');
+const path = require('path');
+
+tsGen.DEFAULT_OPTIONS.bannerComment = `
+/* eslint-disable */
+// prettier-ignore
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+`;
+
+const tsOut = path.resolve('./src/data/form.ts');
+var schemaUri =
```

Comment:
perfect for now ... let's not forget to re-configure this later on :)

Reply:
I think the build will fails as soon as you merge your branch :)